man/sd_journal_seek_head.xml

```diff
@@ -66,46 +66,38 @@
-    ambiguous, it is not defined which position is sought to.</para>
-
-    <para><function>sd_journal_seek_cursor()</function> seeks to the
-    entry located at the specified cursor string. For details on
-    cursors, see
+    <para><function>sd_journal_seek_head()</function> seeks to the beginning of the journal, i.e. to the
+    position before the oldest available entry.</para>
+
+    <para>Similarly, <function>sd_journal_seek_tail()</function> may be used to seek to the end of the
+    journal, i.e. the position after the most recent available entry.</para>
+
+    <para><function>sd_journal_seek_monotonic_usec()</function> seeks to a position with the specified
+    monotonic timestamp, i.e.  <constant>CLOCK_MONOTONIC</constant>. Since monotonic time restarts on every
+    reboot a boot ID needs to be specified as well.</para>
+
+    <para><function>sd_journal_seek_realtime_usec()</function> seeks to a position with the specified
+    realtime (wallclock) timestamp, i.e. <constant>CLOCK_REALTIME</constant>. Note that the realtime clock is
+    not necessarily monotonic. If a realtime timestamp is ambiguous, it is not defined which position is
+    sought to.</para>
+
+    <para><function>sd_journal_seek_cursor()</function> seeks to the position at the specified cursor
+    string. For details on cursors, see
     <citerefentry><refentrytitle>sd_journal_get_cursor</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
-    If no entry matching the specified cursor is found the call will
-    seek to the next closest entry (in terms of time) instead. To
-    verify whether the newly selected entry actually matches the
-    cursor, use
+    If no entry matching the specified cursor is found the call will seek to the next closest entry (in terms
+    of time) instead. To verify whether the newly selected entry actually matches the cursor, use
     <citerefentry><refentrytitle>sd_journal_test_cursor</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
 
-    <para>Note that these calls do not actually make any entry the new
-    current entry, this needs to be done in a separate step with a
-    subsequent
-    <citerefentry><refentrytitle>sd_journal_next</refentrytitle><manvolnum>3</manvolnum></citerefentry>
-    invocation (or a similar call). Only then, entry data may be
-    retrieved via
-    <citerefentry><refentrytitle>sd_journal_get_data</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
-    If no entry exists that matches exactly the specified seek
-    address, the next closest is sought to. If
+    <para>Note that these calls do not actually make any entry the new current entry, this needs to be done
+    in a separate step with a subsequent
     <citerefentry><refentrytitle>sd_journal_next</refentrytitle><manvolnum>3</manvolnum></citerefentry>
-    is used, the closest following entry will be sought to, if
+    invocation (or a similar call). Only then, entry data may be retrieved via
```

Comment:
I would replace "(or a similar call)" with the concrete "(or the sd_journal_previous() call)".

Reply:
but there are more calls like that: sd_journal_previous_skip, sd_journal_next_skip, I didn't want to list them all there...

Reply:
Ok, I wasn't sure if there are more. I still consider the "similar" word in there quite confusing, but I don't know how to write it any better without explicitly naming all the functions. I guess it's ok, then.